generated/thrift/rpc.thrift

```diff
@@ -287,7 +287,7 @@ struct WriteTaggedBatchRawRequest {
 
 struct WriteTaggedBatchRawRequestElement {
 	1: required binary id
-	2: required list<TagRaw> tags
+	2: required binary encodedTags
```

Comment:
Is `TagRaw` used anymore?  If not can we remove it?  And if so maybe we can also rename `TagString` to just `Tag`.

Reply:
sure thing, both done.